Makefile

```diff
@@ -12,12 +12,12 @@ DESTDIR ?= /usr/local
 LOCAL_CFLAGS :=	-Ivendor -Isrc
 FUSE_CFLAGS  :=	$(shell pkg-config --silence-errors --cflags fuse)
 CORE_CFLAGS  := $(shell pkg-config --silence-errors --cflags sqlite3)	\
-		$(shell pkg-config --silence-errors --cflags gmp-6)	\
+		$(shell pkg-config --silence-errors --cflags gmp)	\
 		$(shell pkg-config --silence-errors --cflags re2)	\
 		$(shell pkg-config --silence-errors --cflags-only-I ncurses)
 FUSE_LDFLAGS := $(shell pkg-config --silence-errors --libs fuse    || echo -lfuse)
 CORE_LDFLAGS :=	$(shell pkg-config --silence-errors --libs sqlite3 || echo -lsqlite3)	\
-		$(shell pkg-config --silence-errors --libs gmp-6   || echo -lgmp)	\
+		$(shell echo -lgmp)	\
```

Comment:
Why do you need this change? If `--cflags` worked, wouldn't `--libs` work too?

Reply:
This was just testing what would break without it. (surprisingly nothing)

but for Mac M1 this line actually needs to be 
`$(shell pkg-config --silence-errors --libs gmp   || echo -lgmp)`


ex:
```
> pkg-config --silence-errors --libs gmp-6

> pkg-config --silence-errors --libs gmp 
-L/opt/homebrew/Cellar/gmp/6.2.1_1/lib -lgmp
```

So I was trying to figure out if something like `gmp-6` --fallback--> `gmp` --fallback--> `echo` would work